pkgs/by-name/ge/gemini-cli/package.nix

```diff
@@ -8,18 +8,18 @@
 
 buildNpmPackage (finalAttrs: {
   pname = "gemini-cli";
-  version = "0.1.14";
+  version = "0.1.15";
 
   src = fetchFromGitHub {
     owner = "google-gemini";
     repo = "gemini-cli";
-    tag = "v${finalAttrs.version}";
-    hash = "sha256-u73aqh7WnfetHj/64/HyzSR6aJXRKt0OXg3bddhhQq8=";
+    rev = "pull/5336/head";
```

Comment:
This is not accepted either, the PR can be updated and then this will broke. A PR is not fixed.

Once again, please use fetchpatch and retrieve from that PR each single commit in here.